cloudformation.yaml

```diff
@@ -110,3 +110,17 @@ Resources:
                     Properties:
                         Schedule: cron(12 14 * * ? *) # run daily at 14:12
         DependsOn: LambdaRole
+
+    UnverifiedUserLambda:
+        Type: AWS::Serverless::Function
+        Properties:
+            FunctionName: !Sub gibbons-unverified-cleanup-${Stage}
+            Description: Lambda for cleaning up unverified users and their keys
+            Handler: com.gu.gibbons.lambdas.UnverifiedUserLambda::handleRequest
+            Role: !Sub ${LambdaRole.Arn}
+            Events:
+                ScheduleLambda:
+                    Type: Schedule
+                    Properties:
+                        Schedule: cron(12 14 * * ? *) # run daily at 14:12
```

Comment:
Quick question - do we want to run this at the same time as the other lambdas above? Maybe they should all run at different times actually?

Reply:
I believe both other lambdas are actually running on a different schedule than the one specified in the cloudformation (updated directly in Eventbridge) @JustinPinner